Packs/CommonTypes/ReleaseNotes/3_3_32.md

```diff
@@ -0,0 +1,6 @@
+
+#### Indicator Types
+- **emailRep**
+
```

Comment:
what is that?

Reply:
That's the name for email type (id), it's needed for the validator and added automatically by the formatter.